src/rocm_docs/theme.py

```diff
@@ -62,7 +62,7 @@ def _update_theme_options(app: Sphinx) -> None:
     theme_opts = get_theme_options_dict(app)
     _update_repo_opts(app.srcdir, theme_opts)
 
-    supported_flavors = ["rocm", "local"]
+    supported_flavors = ["rocm", "local", "list"]
```

Comment:
let's make this a little more descriptive.
rocm-api-tools-list?

Reply:
sounds good to me.

done in [refactor: Rename list theme to rocm-api-tools-list](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/539/commits/161fe0658fcac9292a8b911ae0b514051b577e55)